src/client/pages/Payment/Payment.tsx

```diff
@@ -52,7 +52,13 @@ export const PaymentPage = () => {
             <HedvigLogo width={78} />
           </HeaderLogo>
           <HeaderBreadcrumbs>
-            <Breadcrumbs />
+            <Breadcrumbs
+              nextStep={
+                redirectURL?.includes('/confirmation')
```

Comment:
Perhaps we should flip the logic here to make "confirmation" the default

Reply:
Agree, it's probably more common